### PR TITLE
[#166052278] Remove captcha validation

### PR DESCRIPTION
**Pull request by ryanannatuvasquez**
## Description
- Remove captcha validation, it will be no longer need when rejecting a job

### CODE REVIEW

File: src/ApproveRejectValidator.php [hunk 1]
```diff
@@ -102,10 +99,9 @@ protected function validateApprove(array $job)
      */
     protected function validateReject(array $job = array())
     {
-        if (isset($job['reason']) === true && isset($job['comment']) === true && isset($job['captcha']) === true) {
+        if (isset($job['reason']) === true && isset($job['comment']) === true) {
```
Comment (jeanklingler) [inline src/ApproveRejectValidator.php]: (unrelated to your change) maybe you can remove `=== true`?

Comment (ryanannatuvasquez) [reply src/ApproveRejectValidator.php]: @jeanklingler TBH I really liked the idea, however it is a standard everywhere in the whole repository.

Comment (jeanklingler) [reply src/ApproveRejectValidator.php]: OK :+1:

**Comment by ryanannatuvasquez** [comment]
Error in travis not related to the changes. Will address in different PR

**Pull request merged**